content/02.main-text.md

```diff
@@ -432,7 +432,14 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 
 Concepts for the future of scholarly publishing extend beyond collaborative writing [@doi:10.22541/au.148769949.92783646; @doi:10.22541/au.149693987.70506124; @tag:techblog-brown].
 Pandoc Scholar [@doi:10.7717/peerj-cs.112] and Bookdown [@doi:10.1201/9781315204963], which has been used for open writing [@doi:10.12688/f1000research.16516.1], both enhance traditional Markdown to better support publishing.
-The [knitcitations](https://github.com/cboettig/knitcitations) package enables citation by DOI or URL in R Markdown documents.
+[Dokieli](https://dokie.li/) stores articles as HTML and includes a clientside editor to allow reader annotation and interaction [@doi:10.1007/978-3-319-60131-1_33].
+[Texture](https://substance.io/texture/) is a browser-based editor for scholarly manuscripts stored as JATS.
+
+Several projects in addition to Manubot provide infrastructure for citation by identifier. 
+For example, the [knitcitations](https://github.com/cboettig/knitcitations) package enables citation by DOI or URL in R Markdown documents.
+Zotero has developed bibliographic metadata extractors, called [translators](https://github.com/zotero/translators/), for over 500 resources.
+[Citation.js](https://citation.js.org/) converts bibliographic references or identifiers in a variety of formats to CSL JSON [@doi:10.7717/peerj-cs.214].
```

Comment:
Do we want to add that [wcite](https://github.com/wikicite/wcite) wraps it in a Pandoc filter?  That filter along with the [new identifier types](https://github.com/orgs/citation-js/projects/1) being introduced in citation.js are arguably the closest to Manubot's citation-by-identifier.

Reply:
Done in https://github.com/greenelab/meta-review/pull/255/commits/9e42f94ede21c32d7682f3060d9e1b63fd0791f8. @agitter ready for re-review. @nichtich feel free to comment on whether our short mention of pwcite is accurate.

Reply:
Looks good to me.  Let's wait to see whether @nichtich has a comment before merging.

Reply:
Merging now. Comments will be addressed in subsequent PRs as needed.